route/option.go

```diff
@@ -181,9 +182,18 @@ func Grouper(groups [][]string) Option {
 // the `Update()` function. In the `Update()` function you can request a
 // `TimeTracker` from the `state` and use it to get access to time information
 // the route.
-func ValueFunctionMeasures(valueFunctionMeasures []ByIndex) Option {
+func ValueFunctionMeasures(
+	valueFunctionMeasures any,
```

Comment:
why not use `[]any` instead of `any`? Then the compiler can at least verify that the input is a slice.

Reply:
with the change to introduce a new option for dependent value functions/travel time functions, this becomes obsolete.